app/views/RoomsListView/index.js

```diff
@@ -7,6 +7,8 @@ import { connect } from 'react-redux';
 import { isEqual } from 'lodash';
 import { SafeAreaView } from 'react-navigation';
 import Orientation from 'react-native-orientation-locker';
+import moment from 'moment';
+import 'moment/min/locales';
```

Comment:
Is this needed?

Reply:
yes :(
https://github.com/moment/moment/issues/4422